lib/Jit/options.cpp

```diff
@@ -17,19 +33,19 @@ Options::Options(LLILCJitContext *Context) : Context(Context) {}
 
 void Options::initialize() {
   // Set 'IsAltJit' based on environment information.
-  this->setIsAltJit();
+  setIsAltJit();
```

Comment:
Is it *really* in the style guide to remove the "this"?  Not prepending members with 'this->' is like my a #1 pet peeve.